.travis.yml

```diff
@@ -18,10 +14,24 @@ install:
     fi
   - sudo apt-get update -qq -y;
   - sudo apt-get install -qq -y octave liboctave-dev;
-  - "pip install sympy==$SYMPY_VER"
+  - sudo apt-get install -qq -y python;
+  - "pip install --user sympy==$SYMPY_VER"
   - if [ "x$DOCTEST" = "xyes" ]; then
         octave -W --no-gui --eval "pkg install -forge doctest";
     fi
+  - if [ "x$PYTAVE" = "xyes" ]; then
+        sudo apt-get install -qq -y libboost-python-dev python-numpy;
+	# TODO: failures here need to make the tests fail
```

Comment:
@cbm755 For this. Should I run something like:
`octave --path=/home/travis/build/cbm755/octsympy/pytave -W --no-gui --eval "sympref ipc pytave; syms x;"`
before this `if` ends? Will it suffice that octave has error? Or something else?